src/azure-cli/azure/cli/command_modules/iot/commands.py

```diff
@@ -173,3 +178,11 @@ def load_command_table(self, _):  # pylint: disable=too-many-statements
         g.custom_command('create', 'pnp_create_key')
         g.custom_command('delete', 'pnp_delete_key')
         g.custom_command('update', 'pnp_update_key')
+
+    with self.command_group('iot central app', iot_central_sdk, client_factory=iot_central_service_factory) as g:
+        g.custom_command('create', 'iot_central_app_create')
+        g.custom_command('list', 'iot_central_app_list')
+        g.custom_show_command('show', 'iot_central_app_get')
+        g.generic_update_command('update', getter_name='iot_central_app_get',
+                                 setter_name='iot_central_app_update', command_type=update_custom_util)
+        g.custom_command('delete', 'iot_central_app_delete')
```

Comment:
Please add `confirmation=True` for delete command.

Reply:
done

Reply:
I reverted this change because this will break existing scripts customers are using. This will require them to pass in `--yes` in order for it proceeds with existing functionality we had before.

Reply:
Additionally, we really don't want to introduce any new functionality in this PR we just want to migrate existing command module code from `iotcentral` to `iot`

Reply:
Have you add deprecation information for original commands?

Reply:
Yes, the deprecation information for `az iotcentral` command module is in ~/command_modules/iotcentral/commands.py in this PR. That makes sense about breaking them either way, I added it back.